sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -162,39 +162,38 @@ var data = new
 
 Response response = client.AnalyzeConversation(RequestContent.Create(data));
 
-using JsonDocument result = JsonDocument.Parse(response.ContentStream);
-JsonElement conversationalTaskResult = result.RootElement;
-JsonElement conversationPrediction = conversationalTaskResult.GetProperty("result").GetProperty("prediction");
+dynamic conversationalTaskResult = response.Content.ToDynamicFromJson();
+dynamic conversationPrediction = conversationalTaskResult.result.prediction;
 
-Console.WriteLine($"Top intent: {conversationPrediction.GetProperty("topIntent").GetString()}");
+Console.WriteLine($"Top intent: {conversationPrediction.topIntent}");
 
 Console.WriteLine("Intents:");
-foreach (JsonElement intent in conversationPrediction.GetProperty("intents").EnumerateArray())
+foreach (dynamic intent in conversationPrediction.intents)
```

Comment:
Curious: should we prefer `dynamic` or `var` here?  I don't have a strong preference, but I believe either will work, and if we want the code to be substitutable for strongly-based model code later, `var` would give us that where `dynamic` might too, but with increased perf overhead.

Reply:
Our style is set up to dissuade `var` when non-obvious e.g., result of instantiation.